src/Mod/Part/Gui/TaskCheckGeometry.cpp

```diff
@@ -433,11 +433,22 @@ void TaskCheckGeometryResults::goCheck()
     for(const auto &sel :  selection) {
         selectedCount++;
         TopoDS_Shape shape = Part::Feature::getShape(sel.pObject,sel.SubName,true);
-        if (shape.IsNull())
+        if (shape.IsNull()) {
+            Base::Console().Warning(tr("CheckGeometry: Skipping %1, has null or no shape.")
+                                    .arg(QString::fromUtf8(sel.pObject->getNameInDocument())).toStdString().c_str());
+            Base::Console().Message("\n");
```

Comment:
Is this really correct? Shouldn't both be Warning instead of one Warning and one Message? (Or maybe even better, just appended inline)

Reply:
We can dispense with the warnings.  These might confuse the user into thinking there was a problem with the model, anyway.